tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -115,15 +117,21 @@ Clean Up Server
   Wait Until Untitled.ipynb JupyterLab Tab Is Selected
   Close Other JupyterLab Tabs
   Add and Run JupyterLab Code Cell  !rm -rf *
-  
+
 JupyterLab Is Visible
-  ${spawner_visible} =  Run Keyword and Return Status  Wait Until Element Is Visible  xpath:${JL_TABBAR_CONTENT_XPATH}  timeout=20
-  [return]  ${spawner_visible}
+  ${jupyterlab_visible} =  Run Keyword and Return Status  Wait Until Element Is Visible  xpath:${JL_TABBAR_CONTENT_XPATH}  timeout=60
+  [return]  ${jupyterlab_visible}
+
+Wait Until JupyterLab Is Loaded
+  [Arguments]   ${timeout}=60
+  Wait Until Element Is Visible  xpath:${JL_TABBAR_CONTENT_XPATH}  timeout=${timeout}
 
 Clone Git Repository
   [Arguments]  ${REPO_URL}
   # Make sure we are in the root of the server
-  Click Element  xpath://span[@title="/opt/app-root/src"] 
+  Maybe Open JupyterLab Sidebar  File Browser
+  Navigate Home In JupyterLab Sidebar
```

Comment:
I've seen this command failing often in Sanity, probably because the sidebar is not opened:
`Click Element  xpath://span[@title="/opt/app-root/src"]`

`Maybe Open JupyterLab Sidebar  File Browser` makes sure the sidebar is opened in the _File Browser_ tab
`Navigate Home In JupyterLab Sidebar`  is a keyword that Landon made equivalent to `Click Element  xpath://span[@title="/opt/app-root/src"]`

In order to simplify the code I've replaced this 2 keywords for this one: 
`Navigate Home (Root folder) In JupyterLab Sidebar File Browser`

Reply:
New keyword in this commit: https://github.com/red-hat-data-services/ods-ci/pull/72/commits/8b2c323cef18efc21dbaa50cbc783766f427b9cb

Reply:
looks good to me